DEVELOPING.md

```diff
@@ -15,44 +43,115 @@ This project is a monorepo. The different components are:
 _____________
 
-Heroku uses a specific version of node (`12.16.*`). NVM is a tool that lets you easily switch node versions.
-We need this since we want our `local` environment to match `staging` and `prod`
+## Set up environment variables
+
+Before you can start running any of the apps locally, you will need to set up environment variables. You can find them on the [staging Heroku](https://dashboard.heroku.com/apps/crownheightsma-staging/settings) under 'Config Vars'.
+
+Add them in a file named `.env` in the root directory. You can copy it from `.env.example`.
+
+See `app.json` for the definition of each environment variable. You may not need all variables to get started, depending on which part of the app you are working on, but starting with the ones in `.env.example` is a good start.
+
+## Managing Node.js versions
+
+When developing, we want the version of Node.js in our `local` environment to match `staging` and `prod`, which is `12.16.*` as per Heroku's setup.
+
+If you don't already have a version manager, we recommend nvm, a tool that lets you easily switch Node.js versions.
+
+
+### Installing NVM
 
 If you haven't installed nvm, you can just run these commands:
 ```bash
 curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+
 # Verify with:
 command -v nvm
+
 # If you see a number, nvm is now installed!
 # Now install heroku's node version
 nvm install 12.16.1
+
 # Use the version
 nvm use 12.16.1
 ```
 _____________
 
 After you install the correct version of node, install dependencies:
 
-`npm install`
-_____________
+```
+npm install
+```
+
+## Mock data
+We use Airtable as our main data source. If you set up your environment variables right, your apps should automatically start pulling data from our [staging Airtable](https://airtable.com/tblwLlM3yRmIxCYDB/viwbYLZKipJno7jqs), which you should also have access to if you requested it in the [first step](#before-you-start). Feel free to add test data to that Airtable instance.
+
+## Running apps
+All the instructions so far are for general setup for any of the apps in this repo. More likely than not, you will be working on one or two apps only.
+
+If you run into any issues, it is usually due to missing environment variables, so please check those.
+
+### API
+More details in [./src/api/DEVELOPING.md](./src/api/DEVELOPING.md)
+
+```
+npm run local:express
+```
+
+The server is up when you see `Mutual Aid app listening on 5000!`
+
+### Slack bot
+Setting up development for the Slack bot is more involved. Please see [./src/slackapp/DEVELOPING.md](./src/slackapp/DEVELOPING.md)
+
+### Web app
+
+To run the webapp, you will also need to run the API server
+```
+npm run local:react
+npm run local:express
+```
+
+Visiting [http://localhost:3000/delivery-needed](http://localhost:3000/delivery-needed) should show you a delivery requests map.
+
+More details in [./src/webapp/DEVELOPING.md](./src/webapp/DEVELOPING.md)
+
+### Twilio
```

Comment:
@mab253 I don't know enough about setting up Twilio for dev, is linking to the README sufficient or would you like to add a few lines?